<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Personal</class>
 <widget class="QWidget" name="Form_Personal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="BACKGROUND">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QFrame" name="Widget_Main">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>601</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QPushButton" name="btn_setPFP">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>设置头像</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_nickname">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_nickname">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>昵称：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_username">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>账号：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_username">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>180</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_password">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>密码：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_password">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>250</y>
      <width>311</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_signature">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>320</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>签名：</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="te_signature">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>320</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pfp">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: transparent;
border: 1px solid black;
border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="iconSize">
     <size>
      <width>60</width>
      <height>60</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Font: 60pt;</string>
    </property>
    <property name="text">
     <string>我</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_pfp">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>60</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius: 30px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>btn_setPFP</zorder>
   <zorder>le_nickname</zorder>
   <zorder>lbl_nickname</zorder>
   <zorder>lbl_username</zorder>
   <zorder>le_username</zorder>
   <zorder>lbl_password</zorder>
   <zorder>le_password</zorder>
   <zorder>lbl_signature</zorder>
   <zorder>te_signature</zorder>
   <zorder>lbl_title</zorder>
   <zorder>lbl_pfp</zorder>
   <zorder>btn_pfp</zorder>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>510</y>
     <width>201</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>确定修改</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
